web/src/components/ZonesModal/ZonesModal.jsx

```diff
@@ -261,11 +263,13 @@ const ZonesModal = ({
                         setZones();
                         onClose();
                         clearRcaSourceId();
+                        getSystemState();
```

Comment:
Is this another API request? Set zones returns the system state after whatever changes were made, I don't know why this would be necessary.